Dockerfile

```diff
@@ -67,10 +67,10 @@ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys C3173AA6 \
 ENV PYENV_ROOT=/usr/local/.pyenv \
     PATH="/usr/local/.pyenv/bin:$PATH"
 RUN git clone https://github.com/pyenv/pyenv.git /usr/local/.pyenv \
-    && cd /usr/local/.pyenv && git checkout v1.2.17 && cd - \
```

Comment:
I'm concerned about accepting all updates that flow into the `master` branch upstream.

Can we wait for the next release?
If not, could we pin to a specific commit?

Reply:
@thepwagner Fixed.